yaml/src/main/java/com/fasterxml/jackson/jaxrs/yaml/JsonMappingExceptionMapper.java

```diff
@@ -0,0 +1,10 @@
+package com.fasterxml.jackson.jaxrs.yaml;
+
+import javax.ws.rs.ext.Provider;
+
+/**
+ * @deprecated Since 2.2 use {@link com.fasterxml.jackson.jaxrs.base.JsonMappingExceptionMapper} instead.
+ */
+@Deprecated
```

Comment:
Let's not add this -- deprecated for existing modules, so better not to include for new ones, as they can use the shared standard one.